ci-operator/step-registry/cucushift/installer/rehearse/aws/c2s/ipi/disconnected/private/provision/cucushift-installer-rehearse-aws-c2s-ipi-disconnected-private-provision-chain.yaml

```diff
@@ -22,7 +22,6 @@ chain:
     - ref: ipi-install-install-aws
     - ref: ipi-install-times-collection
     - ref: mirror-images-qe-test-images
-    - ref: enable-qe-catalogsource-disconnected
```

Comment:
Why removed it? If this step is removed, all optional operator teams' tests will fail.

Reply:
It's removed temporarily to verify upgrade step, otherwise the testing is being blocked. We will get it recovered before merge.